webapp/sources/rudder/rudder-core/src/main/scala/com/normation/cfclerk/domain/TechniqueVersion.scala

```diff
@@ -37,290 +37,65 @@
 
 package com.normation.cfclerk.domain
 
-import java.nio.charset.Charset
 
-/**
- * When comparing two version numbers, first the epoch of each are compared, then
- * the upstream_version if epoch is equal. epoch is compared numerically. The
- * upstream_version part is compared using the following algorithm :
- *
- * The strings are compared from left to right.
- *
- * First the initial part of each string consisting entirely of non-digit
- * characters is determined. These two parts (one of which may be empty) are
- * compared lexically. If a difference is found it is returned. The lexical
- * comparison is a comparison of ASCII values modified so that all the letters sort
- * earlier than all the non-letters and so that a tilde sorts before anything, even
- * the end of a part. For example, the following parts are in sorted order from
- * earliest to latest: ~~, ~~a, ~, the empty part, a.
- *
- * Then the initial part of the remainder of each string which consists entirely of
- * digit characters is determined. The numerical values of these two parts are
- * compared, and any difference found is returned as the result of the
- * comparison. For these purposes an empty string (which can only occur at the end
- * of one or both version strings being compared) counts as zero.
- *
- * These two steps (comparing and removing initial non-digit strings and initial
- * digit strings) are repeated until a difference is found or both strings are
- * exhausted.
- */
+import com.normation.GitVersion.RevId
+import com.normation.utils.ParseVersion
+import com.normation.utils.Version
+
 
 class TechniqueVersionFormatException(msg: String) extends Exception("The version format of a technique should be : [epoch:]upstream_version\n" + msg)
```

Comment:
this message about epoch surely doesn't make sense, and maybe the whole exception also

Reply:
I think it's not used anymore and need to be removed